From 1984a6003aff83421d4d6f1f1e1d6b4447425e26 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Mon, 18 Mar 2019 11:19:29 +0700
Subject: [PATCH 077/175] arm64: dts: r8a774c0-cat874: Add non-removable
 property

Due to hardware issue on CD pin of micro SD card slot.
Using non-removable property in order to use PLUGGED card on slot.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
index 2908545..72ea412 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
@@ -99,5 +99,6 @@
 	bus-width = <4>;
 	sd-uhs-sdr50;
 	sd-uhs-sdr104;
+	non-removable;
 	status = "okay";
 };
-- 
2.7.4

